testbed/testbed/exporters.go

```diff
@@ -40,33 +51,50 @@ type TraceExporter interface {
 	ProtocolName() string
 }
 
+// TraceExporter defines the interface that allows sending trace data. It adds ability
+// to export a batch of Spans to the Exporter interface.
+type TraceExporter interface {
+	Exporter
+	ExportSpans(spans []*trace.SpanData) error
+}
+
+// MetricExporter defines the interface that allows sending metric data. It adds ability
+// to export a batch of Metrics to the Exporter interface.
+type MetricExporter interface {
+	Exporter
+	ExportMetrics(metrics consumerdata.MetricsData) error
+}
+
+// jaegerExporter implements TraceExporter for Jaeger Thrift-HTTP protocol.
 type jaegerExporter struct {
 	exporter *jaeger.Exporter
 	port     int
 }
 
 // Create a new Jaeger protocol exporter.
 func NewJaegerExporter(port int) *jaegerExporter {
```

Comment:
doc should start with NewJaegerExporter.

Reply:
fixed.